tox.ini

```diff
@@ -20,14 +28,7 @@ commands = pex . -m dql:main -o dist/dql
 [testenv:py38-lint]
 ignore_errors = true
 commands =
-    mypy dql tests setup.py bin/install.py
-    isort --skip snapshots -c -rc dql tests setup.py bin/install.py
-    black --exclude=snapshots --check dql tests setup.py bin/install.py
-    pylint --rcfile=.pylintrc dql tests bin/install.py
-
-[testenv:py36-lint]
-ignore_errors = true
-commands =
+    pip list -l
```

Comment:
Looks unnecessary; was this just added for debugging?

Reply:
It might be un necessary for the pipeline to work, but in my opinion, having it available is very helpful in knowing what might have changed since the last successful job run without having to spend time debuging.